<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>retours</class>
 <widget class="QWidget" name="retours">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>741</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string> ajout d'une station</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>461</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>reference</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>point </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_point">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>type_chargeur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_chargeur">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>temps</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_temps">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierBtn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_phone">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>phone</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>afficher une station</string>
    </attribute>
    <widget class="QTableView" name="tab_stations">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>671</width>
       <height>281</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recherche">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>search...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_trie">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="generate_pdf">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>pdf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="qrcode">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>340</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>390</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="chartview">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>551</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>sms</string>
    </attribute>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>send sms</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="sms">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>321</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>phone number</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>167</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Bureau/login.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
